### DISCUSSION TITLE
ISG or SSG with route rules in RC12

### DISCUSSION THREAD

**Discussion by Mokkapps** [nuxt/content#1626] [General]
What is the correct approach to use SSG or ISG with the new route rules from RC12?

- I don't want to prerender every page, so "nuxi generate" is not an option for me
- Can I use `queryContent` with `useAsyncData` for SSG?
- Should I set `ssr: false`?

I'm very confused

**Comment by Mokkapps** [comment]
I solved it with a similar configuration:

![image](https://user-images.githubusercontent.com/3127210/198012762-fe566421-34e9-469d-aecc-786e5510079a.png)

**Reply by mathijsfr** [reply]
Does it work for you if you change the swr route to prerender?
